packages/flutter/lib/src/services/text_input.dart

```diff
@@ -832,7 +847,11 @@ abstract class TextInputClient {
 ///
 /// See also:
 ///
-///  * [TextInput.attach]
+///  * [TextInput.attach], a method used to establish a [TextInputConnection]
+///    between the system's text input and a [TextInputClient].
+///
```

Comment:
There are a few places with empty lines in between "See also" items like this.  Just wanted to make sure that's intentional.  I don't know if we have an opinion one way or another.

Reply:
Found this in the wiki: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#leave-breadcrumbs-in-the-comments. No empty lines between each items.